test/reports/test-report-manager.js

```diff
@@ -0,0 +1,290 @@
+const mockgoose = require('mockgoose');
+const mongoose = require('mongoose');
+const assert = require('assert');
```

Comment:
The mocha examples I was looking at were using assert, but I'll change to ```Chai``` since that's what we're using.